src/msix/common/AppxFactory.cpp

```diff
@@ -88,7 +88,15 @@ namespace MSIX {
     HRESULT STDMETHODCALLTYPE AppxFactory::CreateBundleWriter(IStream *outputStream, UINT64 bundleVersion, IAppxBundleWriter **bundleWriter) noexcept try
     {
         THROW_IF_BUNDLE_NOT_ENABLED
-        NOTIMPLEMENTED;
+        ThrowErrorIf(Error::InvalidParameter, (outputStream == nullptr || bundleWriter == nullptr || *bundleWriter != nullptr), "Invalid parameter");
+        //#ifdef MSIX_PACK 

```

Comment:
>//#ifdef MSIX_PACK  [](start = 8, length = 19)

commenting this will also make it fail for non pack builds

Reply:
Updated.